### DISCUSSION TITLE
Change settings after the initial setup

### DISCUSSION THREAD

**Discussion by ThomasKoppensteiner** [BitMaker-hub/NerdMiner_v2#355] [Q&A]
Hello,

how can I change the settings, like the pool or the time-zone offset, after the initial setup?

Is there a web view or something similar or do I have to reset the NerdMiner?

**Comment by MyOwn2C** [comment]
Reset it then use QR or AP to reconfigure

**Reply by ThomasKoppensteiner** [reply]
Yes, that is what I did in the end.

It is just not the best UX, especially as the NerdMiner does not automatically adjust to daylight saving time (DST) changes.

**Comment by Strykar** [comment]
Same issue, how would you express `+5:30` UTC?
I tried to set `5.5` during setup but the S3's clock is slower by 30 minutes.

**Reply by MyOwn2C** [reply]
> Same issue, how would you express `+5:30` UTC? I tried to set `5.5` during setup but the S3's clock is slower by 30 minutes.

You cannot.
